data/ecosystem/integrations.yaml

```
@@ -310,3 +310,9 @@
   components: [C++, Java, Go, Python]
   oss: true
   cncf: incubating
+- name: APImetrics
+  url: https://apicontext.com/
+  docsUrl: https://docs.apimetrics.io/docs/export-with-opentelemetry
+  components: [Exporter]
```

would it be possible to know how those exporters are implemented? so are you using opentelemetry collectors, or is this implemented with one of the language SDKs?

---

They're implemented using the Python SDK.

---

```suggestion
  components: [Python]
```

We need to do better in the future on how we distinguish these integrations, but let's have it like that for now.